packages/file-picker/ios/Plugin/Classes/Enums/CustomError.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+public enum CustomError: Error {
+    case directoryNotEmpty
+    case fromMissing
+    case invalidPath
+    case toMissing
+    case unknown
+}
+
+extension CustomError: LocalizedError {
+    public var errorDescription: String? {
+        switch self {
+        case .directoryNotEmpty:
```

Comment:
Let's rename the property to "fileAlreadyExists" and the message to "File already exists." since you pass a file path and not a directory path via "to" (Android and iOS).